### PR TITLE
README.md: How to use the tool with podman

### PR DESCRIPTION
**Pull request by ashcrow**
This came up today so I figured I'd throw it into the ``README.md``.

### CODE REVIEW

**Comment by cgwalters** [comment]
I personally use the "big tools container" pattern, and install rdgo inside it along with a lot of others.  That said I'm fine with this too...let's maybe make it a little more generic by noting that today rdgo relies on `mock` which itself needs `--privileged`?

**Comment by ashcrow** [comment]
@cgwalters updated

File: README.md [hunk 1]
```diff
@@ -107,3 +107,27 @@ is (will become) rather different.
 See also https://fedoraproject.org/wiki/Layered_build_scripts_for_package_maintainers
 for a collection of other projects.
 
+
+### Using this tool as a container
+
+Both `podman` and `docker` are known to work with these instructions.
+
+**Note**: Since this tool uses `mock` any time an image is run as a container it must
+use `--privleged=true`.
```
Comment (cgwalters) [inline README.md]: Typo here, but I just fixed it and pushed rather than round trip.  Thanks for the PR!

**Comment by cgwalters** [comment]
https://github.com/projectatomic/rpmdistro-gitoverlay/commit/8d26a990771f27f3bc44a08d02a7e1ed74adc29b

File: README.md (hunk 1 above)
Comment (ashcrow) [reply README.md]: They call me the human typo machine.

**Pull request closed without merging**